experimental/dsn/arweave/wallet.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
It would be great if you have a README here that
1. explain some of these variables.
2. keep track some of the good links, tutorials that you are reading
3. briefly summarize your thoughts on this network "arweave".